components/proxy/src/main/kotlin/com/hotels/styx/services/YamlFileConfigurationService.kt

```diff
@@ -78,6 +82,10 @@ internal class YamlFileConfigurationService(
                 LOGGER.info("service stopped")
             }
 
+    override fun adminInterfaceHandlers() =
+        mutableMapOf(Pair("origins", TextHttpHandler { originsConfig }))
```

Comment:
I believe you could just:

`        mapOf("origins" to TextHttpHandler { originsConfig })`

Reply:
Thanks. That's a nice construct.
I thought I was having problems with mutable vs immutable earlier, but perhaps it was actually something else.